timeless/auth/auth.py

```diff
@@ -24,10 +32,22 @@ def login(username="", password=""):
 
 
 def forgot_password(email=""):
+    """ Handle the forgot password routine. """
     user = Employee.query.filter_by(email=email).first()
```

Comment:
@ammaratef45 I'd like to see here fail first. 
```
if not email:
   return ...
```

Reply:
@vladarefiev `not email` is never true because it takes a default value of "" if not provided.

Reply:
@ammaratef45 
```
>>> email=""
>>> if not email:
...   print("email is emprt")
... 
email is emprt
```

Reply:
@ammaratef45 FYI